Modules/Loadable/Transforms/Widgets/qMRMLTransformDisplayNodeWidget.cxx

```diff
@@ -552,6 +559,50 @@ void qMRMLTransformDisplayNodeWidget::setContourVisualizationMode(bool activate)
 }
 
 //-----------------------------------------------------------------------------
+void qMRMLTransformDisplayNodeWidget::setInteractive(bool enabled)
```

Comment:
Why not call set EditorVisibility? It makes it hard to follow data and process flow if you suddenly change name of a concept for no good reason.
The best would be to choose a name that is the same everywhere (displayable manager, MRML nodes, preferable even match or be similar to what is displayed to the user).